ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/fees/EIP1559Config.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.core.fees;
+
+public class EIP1559Config {
```

Comment:
These numbers could change in future blocks, especially the target and max gas used.  I think this should be an interface with static methods for each fork.  Call the first one "experimental" or just "EIP1559".  I could see this moving into ProtocolSchedule once it ships, but while it is experimental it's own interface is fine.